code/modules/antagonists/brother/brother.dm

```diff
@@ -89,6 +89,29 @@
 	message_admins("[key_name_admin(admin)] made [key_name_admin(new_owner)] and [key_name_admin(bro)] into blood brothers.")
 	log_admin("[key_name(admin)] made [key_name(new_owner)] and [key_name(bro)] into blood brothers.")
 
+/datum/antagonist/brother/get_admin_commands()
+	. = ..()
+	.["Convert To Traitor"] = CALLBACK(src, .proc/make_traitor)
+
+/datum/antagonist/brother/proc/make_traitor()
+	if(alert("Are you sure? This will turn the blood borther into a traitor with the same objectives!",,"Yes","No") != "Yes")
+		return
+
+	var/datum/antagonist/traitor/tot = new()
+	tot.give_objectives = FALSE
+	
+	for(var/datum/objective/obj in objectives)
+		var/obj_type = obj.type
+		var/datum/objective/new_obj = new obj_type()
+		new_obj.owner = owner
+		new_obj.copy_target(obj)
+		tot.add_objective(new_obj)
+		qdel(obj)
+	objectives.Cut()
```

Comment:
Do you have to cut the blood brothers objective list if they will just be cleared when it's removed?

Reply:
If I don't both sets of objectives will be listed when the traitor datum is added